Composer/packages/client/src/ShellApi.ts

```diff
@@ -89,6 +88,7 @@ export const ShellApi: React.FC = () => {
     apiClient.registerApi('createLgFile', ({ id, content }, event) => fileHandler(LU, CREATE, { id, content }, event));
     apiClient.registerApi('updateLgTemplate', updateLgTemplateHandler);
     apiClient.registerApi('removeLgTemplate', removeLgTemplateHandler);
+    apiClient.registerApi('removeLgTemplates', removeLgTemplatesHandler);
```

Comment:
will you also register copyTemplate here?

Reply:
sure